puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/release.groovy

```diff
@@ -12,6 +12,12 @@ void push_rpms_direct(repo_source, repo_dest, overwrite = true, merge = false) {
     }
 }
 
+void push_rpms_katello(version) {
+    sshagent(['katello-fedorapeople']) {
+        sh "ssh katelloproject@fedorapeople.org \"cd /project/katello/bin && sh rsync-repos-from-koji ${version}\""
```

Comment:
```suggestion
        sh "ssh katelloproject@fedorapeople.org 'cd /project/katello/bin && sh rsync-repos-from-koji ${version}'"
```

Reply:
Why is this needed? I'm not quite sure where the variable interpolation happens.

Reply:
`version` is a groovy var here, groovy replaces vars in double quotes, so what the shell sees doesn't have any vars and does not need to be in double quotes.

Reply:
also, it's not *needed*, I just thought not having to escape the " is nice

Reply:
Updated